docker/worker/Dockerfile

```diff
@@ -17,6 +18,7 @@ RUN apt-get update && apt-get -y install \
     software-properties-common \
     sudo \
     testdisk \
+    unzip \
```

Comment:
What's this for?  The upgrader maybe?

Reply:
Unzipping the release we got from Github

Reply:
Oh, which I just realised is a tar.  This is leftover, I think